frontend/src/components/DashboardTable.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+  <v-card flat :color="colour" class="mt-2" height="100%">
+    <v-row class="pt-4 px-8">
+      <v-col class="pa-0" cols="2">
+        <v-card-title class="pa-0">
+          <h3>
+            <v-row no-gutters class="dashboard-title mr-4">{{ title }}</v-row>
+          </h3>
+        </v-card-title>
+      </v-col>
+      <v-col v-for="(row, index) in tableData" :key="index" class="py-0" cols="3">
+        <v-row class="pa-0">
+          <h3>{{ row.title }}</h3>
+        </v-row>
+        <v-row v-for="(col, idx) in omit(row, 'title')" :key="idx" class="pt-2 listCol">
+          <div v-if="idx === 'heldReview' && col.data > 0">
+            <router-link to="heldRequestBatch">
+              <strong>{{ col.data }} {{ col.name }}</strong>
+            </router-link>
+          </div>
```

Comment:
Lines 16 - 20 can be removed. It just makes a link when there are fixable records in a batch file. This doesn't exist on the school side.

Reply:
commented codes removed